demos/src/yaml.mustache

```diff
@@ -0,0 +1,42 @@
+<div class="demo" data-o-component='o-syntax-highlight'>
+	<pre>
+		<code class='o-syntax-highlight--yaml'>
```

Comment:
It would be useful to have examples for more niche yaml features, e.g:

- start tags
	 ```yaml
	---
	```
- anchors: 
	```yaml
		---
	```
- multiline strings (`>` | `|` or the few other cases):
	```yaml
	some_string: |
		my string
		over two lines
	```
- json in yaml

Reply:
Good example of syntax I found: https://learnxinyminutes.com/docs/yaml/

Reply:
![](https://user-images.githubusercontent.com/1569131/61527840-43339580-aa15-11e9-8453-846debd4a819.png)